packages/oidf-client/__tests__/shared/oidfClientAgentLogic.ts

```diff
@@ -1,46 +1,31 @@
 import 'cross-fetch/polyfill'
```

Comment:
Can it still find this? "cross-fetch": "^3.1.8" is not in devDependencies anymore. 
(Maybe do "pnpm prune" followed by "pnpm build:clean" then run the test to confirm.